cdap-cli/src/main/java/co/cask/cdap/cli/english/Article.java

```diff
@@ -39,4 +40,13 @@ public String toString(Word word) {
     }
   }
 
+  /**
+   * "the"
+   */
+  private static class The extends Article {
```

Comment:
Some methods invoke `Fragment.of(Article.A...`. For consistency, I introduced `Article.THE` which can be used in the same way but it's not necessary. I can cut it if you think it's clearer.